Fred2/EpitopeAssembly/EpitopeAssembly.py

```diff
@@ -93,7 +93,7 @@ def __init__(self, peptides, pred, solver="glpk", weight=0.0, matrix=None, verbo
             cleave_pred = pred.predict(list(fragments.keys()))
             #cleave_site_df = cleave_pred.xs((slice(None), (cleavage_pos-1)))
             for i in set(cleave_pred.index.get_level_values(0)):
-                fragment = "".join(cleave_pred.ix[i]["Seq"])
+                fragment = "".join(cleave_pred.loc[i]["Seq"])
```

Comment:
AttributeError: 'CleavageSitePredictionResult' object has no attribute 'ix'
-> ix deprecated in pandas; update to loc
https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#ix-indexer-is-deprecated